mock/mock.go

```diff
@@ -148,6 +148,12 @@ func (c *Call) Times(i int) *Call {
 	c.lock()
 	defer c.unlock()
 	c.Repeatability = i
+
+	// Add expected calls to the parent mock.
+	for j := 1; j < i; j++ {
+		c.Parent.ExpectedCalls = append(c.Parent.ExpectedCalls, c)
```

Comment:
`c` is a `*Call` (pointer to Call), so you actually casue the same call to be added to the Mock's expectations `i` times.

This is working the same way it used to because we are decrementing `c.Repeatabilty` on all the expectations at once, since they are the same expectation. The only difference is that the error now says you need to make `i` more calls, rather than "1" more calls, regarless of how many more you actually need to make.